src/components/Button/ArrowButton/styles.ts

```diff
@@ -1,54 +1,72 @@
-import styled, { css } from "styled-components";
+import styled, { css, RuleSet } from 'styled-components';
+
 import {
-    buttonSizes,
-    buttonVariants,
-    buttonColors,
-    defaultButtonSize,
-} from "@/assets/styles/constants/buttonConstants";
-import { COLORS } from "@/assets/styles/constants/colors";
-import { FONTS } from "@/assets/styles/constants/fonts";
-import { ButtonSize } from "@/assets/styles/constants/buttonConstants";
+  buttonSizes,
+  buttonVariants,
+  buttonColors,
+  defaultButtonSize,
+  ButtonSize,
+} from '@/assets/styles/constants/buttonConstants';
+import { COLORS } from '@/assets/styles/constants/colors';
+import { FONTS } from '@/assets/styles/constants/fonts';
 
 type ButtonVariantType = keyof typeof buttonVariants;
 type ButtonColorType = keyof typeof buttonColors;
 
+// Constants for opacity and transition
```

Comment:
I think this comment useless, by the name of the variable its clear what they done